hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/footer/DeleteActionsLayout.java

```diff
@@ -311,15 +305,10 @@ private boolean isDsInUseInBulkUpload(final Set<DistributionSetIdName> distribut
         return false;
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
Same moving the unchecked @SuppressWarnings("unchecked") to line `310` the only unchecked operation in the method.